Index: google/atom/__init__.py
===================================================================
--- google/atom/__init__.py	(revision 427)
+++ google/atom/__init__.py	(working copy)
@@ -275,7 +275,11 @@
     for xml_attribute, member_name in self.__class__._attributes.iteritems():
       member = getattr(self, member_name)
       if member is not None:
-        tree.attrib[xml_attribute] = member.decode(MEMBER_STRING_ENCODING)
+        if not isinstance(member, unicode):
+          tree.attrib[xml_attribute] = member.decode(MEMBER_STRING_ENCODING)
+        else:
+          tree.attrib[xml_attribute] = member
+
     # Lastly, call the ExtensionContainers's _AddMembersToElementTree to 
     # convert any extension attributes.
     ExtensionContainer._AddMembersToElementTree(self, tree)
